src/features/api/lib/client.ts

```diff
@@ -7,8 +7,10 @@ import type {
 } from './network-provider'
 import { NetworkProvider } from './network-provider'
 
-const apiUrl = env('NEXT_PUBLIC_API_URL')
-const apiKey = env('NEXT_PUBLIC_API_KEY')
+const config = {
+  url: env('NEXT_PUBLIC_API_URL'),
+  key: env('NEXT_PUBLIC_API_KEY'),
+}
```

Comment:
By grouping some configurations or similar variables together we can easily add valuable meaning to the group, and many times simplify how these values are read.